cvat/apps/documentation/user_guide.md

```diff
@@ -1,232 +1,442 @@
+- [User's guide](#users-guide)
+  - [Getting started](#getting-started)
+    - [Authorization](#authorization)
+    - [Administration panel](#administration-panel)
+    - [Creating an annotation task](#creating-an-annotation-task)
+    - [Model manager](#model-manager)
+    - [Search](#search)
+  - [Interface of the annotation tool](#interface-of-the-annotation-tool)
+    - [Basic navigation](#basic-navigation)
+    - [Types of Shapes (basics)](#types-of-shapes-basics)
+    - [Annotation mode (basics)](#annotation-mode-basics)
+    - [Interpolation mode (basics)](#interpolation-mode-basics)
+    - [Attribute annotation mode (basics)](#attribute-annotation-mode-basics)
+    - [Downloading annotations](#downloading-annotations)
+    - [Task synchronization with a repository](#task-synchronization-with-a-repository)
+    - [Vocabulary](#vocabulary)
+    - [Workspace — Context menu](#workspace--context-menu)
+    - [Settings](#settings)
+    - [Bottom Panel](#bottom-panel)
+    - [Side panel](#side-panel)
+      - [Objects](#objects)
+      - [Labels](#labels)
+    - [Bottom side panel](#bottom-side-panel)
+    - [Fullscreen player mode](#fullscreen-player-mode)
+  - [Annotation mode (advanced)](#annotation-mode-advanced)
+  - [Interpolation mode (advanced)](#interpolation-mode-advanced)
+  - [Attribute annotation mode (advanced)](#attribute-annotation-mode-advanced)
+  - [Annotation with polygons](#annotation-with-polygons)
+  - [Annotation with polylines](#annotation-with-polylines)
+  - [Annotation with points](#annotation-with-points)
+  - [Annotation with Auto Segmentation](#annotation-with-auto-segmentation)
+  - [Shape grouping](#shape-grouping)
+  - [Filter](#filter)
+  - [Analytics](#analytics)
+  - [Shortcuts](#shortcuts)
+  - [Hints](#hints)
+
 # User's guide
 
-Computer Vision Annotation Tool (CVAT) is a web-based tool which helps to annotate video and images for Computer Vision algorithms. It was inspired by [Vatic](http://carlvondrick.com/vatic/) free, online, interactive video annotation tool. CVAT has many powerful features: __interpolation of bounding boxes between key frames, automatic annotation using TensorFlow OD API, shortcuts for most of critical actions, dashboard with a list of annotation tasks, LDAP and basic authorization, etc...__ It was created for and used by a professional data annotation team. UX and UI were optimized especially for computer vision tasks developed by our team.
+Computer Vision Annotation Tool (CVAT) is a web-based tool which helps to
+annotate video and images for Computer Vision algorithms. It was inspired
+by [Vatic](http://carlvondrick.com/vatic/) free, online, interactive video
+annotation tool. CVAT has many powerful features: _interpolation of bounding
+boxes between key frames, automatic annotation using TensorFlow OD API,
```

Comment:
There is automatic annotation not only with TF OD API

Reply:
What else should we right here?

Reply:
For example: "automatic annotation using deep learning models"